cmplibrary/src/main/java/com/sourcepoint/cmplibrary/unity3d/UnityUtils.kt

```diff
@@ -0,0 +1,38 @@
+@file:JvmName("UnityUtils")
+
+package com.sourcepoint.cmplibrary.unity3d
+
+import com.sourcepoint.cmplibrary.SpConsentLib
+import com.sourcepoint.cmplibrary.model.exposed.toJsonObject
+
+fun throwableToException(throwable: Throwable) {
+    throw java.lang.Exception(throwable)
+}
+
+fun <T> arrayToList(array: Array<T>): List<T> {
+    return array.toList()
+}
+
+fun callCustomConsentGDPR(
+    spLib: SpConsentLib,
+    vendors: Array<String>,
+    categories: Array<String>,
+    legIntCategories: Array<String>,
+    successCallback: UnityCustomConsentGDPRProxy
+) {
+    val vendorsList = arrayToList(vendors)
+    val categoriesList = arrayToList(categories)
+    val legIntCategoriesList = arrayToList(legIntCategories)
+    spLib.customConsentGDPR(
+        vendorsList, categoriesList, legIntCategoriesList,
+        success = { spCustomConsents ->
+            println("Kotlin-side custom consent: [$spCustomConsents]")
+            if (spCustomConsents != null) {
```

Comment:
It was debug log to test communication between aar artefact and unity sdk. I suggest this conversation resolved since function was refactored in SpConsentLibImpl.kt; it has no println anymore